src/components/Header.js

```diff
@@ -86,6 +87,24 @@ export default function Header() {
           <option value="publishedAt">published at</option>
         </select>
       </div>
+      <button
+        className={styles["header__favorites"]}
+        onClick={() => {
+          if (JSON.parse(localStorage.getItem("favorites")).length) {
```

Comment:
If I understood it correctly. If user doesn't have any favorites it's not possible to visit the page? If I'm right, then I'd like to change this behaviour to switch anyway to the favorites page and show something like 'you don't have any favorites news'